app/views/resources/_upload_localized_resource_template.haml

```diff
@@ -28,6 +28,12 @@
           %input{:type => 'file', 'data-bind' => 'fileupload: url, fileuploadAdd: add, fileuploadprogressall: showProgress'}
         Replace
       / /ko
+      / ko if: hasAudio() && (uploadStatus() == 'ok' || uploadStatus() == 'standBy')
+      %br
+      %audio{'controls' => true, "style" => "width: 450px", "preload" => "none"}
```

Comment:
IE9+ supports `audio` tag. And the call flow designer currently does not work in IE8:

![screen shot 2016-11-04 at 11 34 53 am](https://cloud.githubusercontent.com/assets/429604/20009363/fed24e82-a282-11e6-82ff-757ca8fec0d1.png)